ldclient/impl/events/event_processor.py

```diff
@@ -44,6 +44,7 @@ class DebugEvent:
     def __init__(self, original_input: EventInputEvaluation):
         self.original_input = original_input
 
+
```

Comment:
Lots of little whitespace changes while I was in this file. You can ignore pretty much everything except the 3 lines I'll call out.